ads/google/doubleclick.js

```diff
@@ -100,14 +100,12 @@ function doubleClickWithGpt(global, data, gladeExperiment) {
     const primaryWidth = primarySize[0];
     const primaryHeight = primarySize[1];
     // By default, this flag is true.
-    const multiSizeValidation = data.multiSizeValidation ?
-        data.multiSizeValidation == 'true' : true;
 
     getMultiSizeDimensions(
         multiSizeDataStr,
         primaryWidth,
         primaryHeight,
-        multiSizeValidation,
+        (data.multiSizeValidation || 'true') == 'true',
```

Comment:
Yeah, the issue is that `data` is typed as "Object", so the compiler makes no assumptions about its contents, nor should it since this particular parameter may in fact be omitted.